datafusion/core/tests/sqllogictests/test_files/array.slt

```diff
@@ -1009,48 +1063,117 @@ NULL 1 1
 2 NULL 1
 2 1 NULL
 
-## array_contains
```

Comment:
This looks to me like the tests for `array_contains` have been removed. Was that by accident?

Reply:
No, I `array_contains` is deprecated, the logic is not the same as `array_has_all` or `array_has`.